collections/VideoCollection.js

```diff
@@ -0,0 +1,10 @@
+//Create a video collection that holds the video models of data
+// 
+// var query = 'cats';
+//
+// var APIkey = 'AIzaSyCRxZDF8wrnm67fIS3QkhUXhJMuIWOUMco';
+
+var VideoCollection = Backbone.Collection.extend({
+  model: VideoModel
```

Comment:
Here is where the fetch should have happened.  Since the data coming back from the server is a list of videos, your collection should have been the jumping off point for the fetching of data.